source/_patterns/00-atoms/images/logo.mustache

```diff
@@ -0,0 +1 @@
+<img src="/images/logo-wordmark-white.png" data-rjs="3" class="c-logo" alt="{{ company.name }}" />
```

Comment:
replace with src="../../../images/logo-wordmark-white.png" currently the src is pointing to the images inside atoms.